magpie/nn/models.py

```diff
@@ -12,8 +12,6 @@ def get_nn_model(nn_model, embedding, output_length):
         return cnn(embedding_size=embedding, output_length=output_length)
     elif nn_model == 'rnn':
         return rnn(embedding_size=embedding, output_length=output_length)
-    elif nn_model == 'crnn':
-        return crnn(embedding_size=embedding, output_length=output_length)
```

Comment:
Can you add in the commit log why `crnn` is obsolete? Is there some online reference you can point to?